api/numaresourcesoperator/v1alpha1/numaresourcesoperator_types.go

```diff
@@ -104,19 +107,22 @@ type NUMAResourcesOperatorStatus struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=status,displayName="RTE MCPs from node groups"
 	MachineConfigPools []MachineConfigPool `json:"machineconfigpools,omitempty"`
 	// Conditions show the current state of the NUMAResourcesOperator Operator
+	//+operator-sdk:csv:customresourcedefinitions:type=status,displayName="Condition reported"
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
 }
 
 // MachineConfigPool defines the observed state of each MachineConfigPool selected by node groups
 type MachineConfigPool struct {
 	// Name the name of the machine config pool
 	Name string `json:"name"`
-
 	// Conditions represents the latest available observations of MachineConfigPool current state.
 	// +optional
+	//+operator-sdk:csv:customresourcedefinitions:type=status,displayName="Optional conditions reported for this NodeGroup"
```

Comment:
are we tied to operator-sdk here and in future versions ?

Reply:
could you elaborate please?

Reply:
using operator-sdk frameowrk here ties us to the API it provides (which is ok)  
In case we will want to break the scheme and move to a more native one then it will require refactoring. 
I dont think this will be the case here so we should be fine.